### PR TITLE
Add some externs

### PR DESCRIPTION
**Pull request by oterral**

### CODE REVIEW

File: Cesium.externs.js [hunk 1]
```diff
@@ -2763,6 +2768,11 @@ Cesium.CustomDataSource.prototype.entities;
  */
 Cesium.EntityCollection = function() {}
 
+/**
+ * @param {string} id
```
Comment (gberaudo) [inline Cesium.externs.js]: According to Cesium code, it looks to me that it should be `String|Number`. I opened a tentative fix in Cesium, see https://github.com/AnalyticalGraphicsInc/cesium/pull/5791.

Comment (oterral) [reply Cesium.externs.js]: Yes makes sense.

**Comment by gberaudo** [comment]
Let's see the outcome of the Cesium PR before merging.

File: Cesium.externs.js (hunk 1 above)
Comment (gberaudo) [reply Cesium.externs.js]: It appears your previous attempt was the good one. Can you revert to it?

Comment (oterral) [reply Cesium.externs.js]: Done

**Comment by gberaudo** [comment]
Thanks!

**Pull request merged**